src/Symfony/Component/Validator/Resources/translations/validators.zh_TW.xlf

```diff
@@ -402,6 +402,30 @@
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
                 <target>網絡掩碼的值應當在 {{ min }} 和 {{ max }} 之間。</target>
             </trans-unit>
+            <trans-unit id="104">
+                <source>The filename is too long. It should have {{ filename_max_length }} character or less.|The filename is too long. It should have {{ filename_max_length }} characters or less.</source>
+                <target>該檔名長度太長，長度不可超過 {{ filename_max_length }} 個字元。</target>
```

Comment:
Are there no singular/plural variants in Chinese?

Reply:
Yes. Chinese does not have a distinct singular/plural. Chinese normally use quantifiers, or numbers to indicate whether something is singular or plural.

So I refer to `<trans-unit id="19">` target value